lib/bulk/common.js

```diff
@@ -20,23 +20,6 @@ const INSERT = 1;
 const UPDATE = 2;
 const REMOVE = 3;
 
-const bson = new BSON([
```

Comment:
this reminds me, we'll need to take a look at the `bson-ext` package to make sure we support bson v4 over there. It wasn't obvious to me originally, but the reason for this constructor is to make the types accessible in the C++ addon, so it can instantiate them during serialization/deserialization.

Reply:
I've looked into `bson-ext`. I think we should hold off on porting that library, it's likely that we should take a more dramatic approach there and immediately covert it to N-API given the API changes are non-trivial.